src/assets/docData.json

```diff
@@ -0,0 +1,23 @@
+{
+    "gettingStarted": [{
```

Comment:
change it to `getting-started` so that it will get mapped properly with the name appearing into the route